concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -2014,6 +2020,44 @@ jobs:
             params:
               file: updated-tfstate/datadog.tfstate
 
+        - task: deploy-cve-notifier
+          config:
+            platform: linux
+            image_resource:
+              type: docker-image
+              source:
+                repository: governmentpaas/cf-cli
+            inputs:
+              - name: cve-notifier
+              - name: paas-cf
+              - name: config
+              - name: bosh-CA
+            params:
+              ENABLE_CVE_NOTIFIER: {{enable_cve_notifier}}
+            run:
+              path: sh
+              args:
+                - -e
+                - -c
+                - |
+                  if ! $ENABLE_CVE_NOTIFIER; then
```

Comment:
:-/ ... not sure about this. You are actually executing the variable ${ENABLE_CVE_NOTIFIER}.

Can you better do: `if [ "${ENABLE_CVE_NOTIFIER}" == "true" ]; then`